docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -21,7 +21,9 @@ repave schedule and methodology. Refer to
 
 :::
 
-## Limitations
+## Update a Cluster
+
+### Limitations
```

Comment:
This Limitations section needs to be moved above **Update a Cluster** and changed to a header 2. This is because it's relevant to both How-tos in this article.